transports/webrtc-utils/README.md

```diff
@@ -0,0 +1,6 @@
+# WebRTC Common Utilities
+
+Tools that are common to more than one WebRTC component.
+
+-   [Protobuf Generated Code](generated)
+-   [Stream State](substream/state.rs)
```

Comment:
```suggestion
-   [Protobuf Generated Code](src/generated)
-   [Stream State](src/substream/state.rs)
```

Otherwise these are dead links, right?

Reply:
Yes, very true! Fixed.

I wasn't quite sure what to put in a `utils` readme yet, and Thomas was suggesting making the public API somewhat opaue which hasn't been done yet. Something we are still kind of working through